pkg/kvstore/trace.go

```diff
@@ -17,8 +17,8 @@ func EnableTracing() {
 }
 
 // Trace is used to trace kvstore debug messages
-func Trace(format string, err error, fields logrus.Fields, a ...interface{}) {
+func Trace(format string, err error, fields logrus.Fields, args ...interface{}) {
```

Comment:
I'd personally drop the `args` argument altogether (changing `Debugf` to `Debug`), considering that it is never used, and that `fields` already serves the same purpose.

Reply:
Thanks, adjusted accordingly.